docs/basic-features/environment-variables.md

```diff
@@ -147,3 +147,29 @@ export default async () => {
   loadEnvConfig(projectDir)
 }
 ```
+
+## Environment Variable Load Order
+
+Depending on the environment (as set by `NODE_ENV`), variables are loaded from the following sources (last one wins):
```

Comment:
"last one wins" my be confusing to the reader because it sounds like the last file wins.

Its more of a [union](https://en.wikipedia.org/wiki/Union_(set_theory)) between the environment variables in each of these files.